docs/developer-guide/initialising-modules.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Initialising modules
+section: Modules
+permalink: /docs/developer-guide/modules/initialising-modules/
+site_section: developer-guide
+---
+
+# Initialising modules
+
+_Build Service users don't need to do any JavaScript initialisation. You only need to do this if you're using a manual build process. [Why?](#note-build-service)_
+
+<aside class="o-techdocs__aside--toggleable" id="note-build-service">
+	<h4>Initialising JS and the Build Service</h4>
+	<p>All JavaScript bundles from the Build Service include a module called <code>o-autoinit</code>. <code>o-autoinit</code> does the initialisation step automatically so you don't have to do it yourself. Visit <a href='http://registry.origami.ft.com/components/o-autoinit'>o-autoinit on the Registry</a>.</p>
+</aside>
+
+
+Individual Origami module JavaScript does not perform any initialisation automatically. ie, putting the JavaScript and HTML for `o-gallery` on a page is not enough to get a working `o-gallery` (unless you're using the Build Service, which will auto-initialise components).
```

Comment:
Good call on the readability, your suggestion is much easier to parse. 

Going to leave the build service mention in there, my concern is that people skim read stuff so may have missed it before.